its/ruling/src/test/java/org/sonar/php/it/PhpRulingTest.java

```diff
@@ -34,17 +33,12 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class PHPRulingTest {
+public class PhpRulingTest {
```

Comment:
I can imagine it can cause problems on Windows by git pull

Reply:
It's better to rename it in 2 steps e.g.: `PHPRulingTest` -> `PhpRulingTest2` -> `PhpRulingTest`